### PR TITLE
Module timeout

### PR DESCRIPTION
**Pull request by thefurfel**
Added timeout for `CheckRunnable` and fixed a bug where it could loop infinitely.

### CODE REVIEW

File: Manager/src/main/java/io/raspberrywallet/manager/modules/Module.java [hunk 1]
```diff
@@ -1,5 +1,7 @@
 package io.raspberrywallet.manager.modules;
 
+import org.omg.CORBA.TIMEOUT;
```
Comment (thefurfel) [inline Manager/src/main/java/io/raspberrywallet/manager/modules/Module.java]: Needs to be removed in the future (unused import, thanks jetbrains)

**Review by stanbar — CHANGES_REQUESTED** [review]
Please English comments, it looks bad

**Comment by stanbar** [comment]
Are you going to change those comments? This PR is hanging for almost week 🤣

**Comment by PatrykMilewski** [comment]
@Furfel

**Comment by pfoof** [comment]
Lol, I changed the comments in some other push .-.

**Comment by PatrykMilewski** [comment]
@Furfel there is still one, small conflict

**Review by stanbar — APPROVED** [review]
Good job, keep it up !

**Pull request merged**